client/session.go

```diff
@@ -1224,79 +1246,88 @@ func (s *session) FetchBlocksMetadataFromPeers(
 	var (
 		metadataCh = make(chan blocksMetadata, blocksMetadataChannelInitialCapacity)
 		errCh      = make(chan error, 1)
-		m          = s.streamFromPeersMetricsForShard(shard, resultTypeMetadata)
+		m          = s.newPeerMetadataStreamingProgressMetrics(shard, resultTypeMetadata)
 	)
 
 	go func() {
-		errCh <- s.streamBlocksMetadataFromPeers(namespace, shard, peers,
-			start, end, metadataCh, m)
+		errCh <- s.streamBlocksMetadataFromPeers(
+			namespace, shard, peers, start, end, metadataCh, m, FetchBlocksMetadataEndpointV1)
```

Comment:
@robskillington Its ok to just leave this as V1 only right?

Reply:
Yup sounds good, we can update this code path when we deprecate V1.